modules/nodes-cluster-resource-override-move-infra.adoc

```diff
@@ -0,0 +1,188 @@
+// Module included in the following assemblies:
+//
+// * nodes/clusters/nodes-cluster-overcommit.adoc
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+ifeval::["{context}" == "nodes-cluster-overcommit"]
+:cro:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-cluster-resource-override-move-infra_{context}"]
+= Moving the Cluster Resource Override Operator pods
+
+By default, the Cluster Resource Override Operator installation process creates an Operator pod and two Cluster Resource Override pods on nodes in the `clusterresourceoverride-operator` namespace. You can move these pods to other nodes, such as infrastructure nodes, as needed.
+
+ifdef::cro[]
+You can create and use infrastructure nodes to host only infrastructure components, such as the default router, the integrated container image registry, and the components for cluster metrics and monitoring. These infrastructure nodes are not counted toward the total number of subscriptions that are required to run the environment. For more information about infrastructure nodes, see "Creating infrastructure machine sets."
```

Comment:
```suggestion
You can create and use infrastructure nodes to host only infrastructure components, such as the default router, the integrated container image registry, and the components for cluster metrics and monitoring. These infrastructure nodes are not counted toward the total number of subscriptions that are required to run the environment. For more information about infrastructure nodes, see "Creating infrastructure machine sets".
```

Reply:
@dfitzmau Thanks. I need a new ISG. The version I have shows _see "Creating infrastructure machine sets."_ is correct!